src/main/jbake/content/docs/components/metrics/index.adoc

```diff
@@ -1258,7 +1258,380 @@ A reset event occurs some time between 90000 and 210000; consequently, we will g
 Note that we exclude the rate data point between 90000 and 210000 timestamps.
 
 === Downsampling
-*TODO*
+Downsampling is a query technique for reducing the number of data points that are sent back to the client. Why is this
+done? When a request is made to render a graph, the client specifies a date range. The number of data points that fall
+within that range can and will vary. We want to avoid sending back too many data points because an excessive number of
+data points does little to improve the visualization, slows down the rendering, and makes the UI less response which in
+turn makes the user experience worse overall. Downsampling is a way to return a predictable or fixed number of data
+points which facilitates better graphs and a better overall user experience.
+
+Hawkular Metrics provides several `/stats` endpoints that use downsampling.
```

Comment:
maybe indicate that these /stats endpoints apply to the different type of metrics?